client/starwhale/base/uri.py

```diff
@@ -216,7 +216,11 @@ def real_request_uri(self) -> t.Union[str, Path]:
 
     @property
     def sw_instance_config(self) -> t.Dict[str, t.Any]:
-        return self._sw_config.get_sw_instance_config(self.instance)
+        return (
+            self._sw_config.get_sw_instance_config(self.instance)
```

Comment:
How do we define the situation when a user uses the same URL with a different token?
cc @tianweidut

Reply:
a URL + a token = a instance

Reply:
a URL + a token == an instance alias?

Reply:
if user write uri with the foramt of '${instance URL}/project/...', we can only filter the instance from config by url, but if the format is '${instance-alias}/project/...', We just to use alias to find exactly what the user is referring to。